File: Sandbox.Modules.FSharp/DemoModule.fs
namespace Sandbox.Modules.FSharp

open Sandbox.Core

[<SandboxModule("F# Demo Module", "fs-demo", "This module is a simple test of integrating the Visual Basic module collection with the module service.")>]
type DemoModule() =
   inherit SandboxModule()
   override b.Execute() = printfn "Hello World from an F# module!"

File: Sandbox.Modules.VisualBasic/DemoModule.vb
Imports Sandbox.Core

<SandboxModule("Demo Module", "vb-demo", "This module is a simple test of integrating the Visual Basic module collection with the module service.")>
Public Class DemoModule
    Inherits SandboxModule

    Protected Overrides Sub Execute()
        SendResponse(SandboxEventType.None, "This is a test of integrating a Visual Basic module!")
    End Sub
End Class
